# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1386014245 18000
#      Mon Dec 02 14:57:25 2013 -0500
# Branch stable
# Node ID 62f94ef938b48a1e477fa7bc3e781aecb61e21b4
# Parent  708173343c5059201b7ec7bdbd44aa5fee519f21
don't use period after end of @seealso{...}

* file-io.cc (FSEEK_CUR, FSEEK_END): Don't use period after @seealso
in doc string.

diff --git a/libinterp/corefcn/file-io.cc b/libinterp/corefcn/file-io.cc
--- a/libinterp/corefcn/file-io.cc
+++ b/libinterp/corefcn/file-io.cc
@@ -2217,28 +2217,28 @@ Position file relative to the end.\n\
   return const_value ("SEEK_SET", args, -1);
 }
 
 DEFUNX ("SEEK_CUR", FSEEK_CUR, args, ,
         "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {} SEEK_CUR ()\n\
 Return the numerical value to pass to @code{fseek} to\n\
 position the file pointer relative to the current position.\n\
-@seealso{SEEK_SET, SEEK_END}.\n\
+@seealso{SEEK_SET, SEEK_END}\n\
 @end deftypefn")
 {
   return const_value ("SEEK_CUR", args, 0);
 }
 
 DEFUNX ("SEEK_END", FSEEK_END, args, ,
         "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {} SEEK_END ()\n\
 Return the numerical value to pass to @code{fseek} to\n\
 position the file pointer relative to the end of the file.\n\
-@seealso{SEEK_SET, SEEK_CUR}.\n\
+@seealso{SEEK_SET, SEEK_CUR}\n\
 @end deftypefn")
 {
   return const_value ("SEEK_END", args, 1);
 }
 
 static octave_value
 const_value (const char *, const octave_value_list& args,
              const octave_value& val)
